### PR TITLE
Enable create secret for deepspeedjob, etjob.

### PR DESCRIPTION
**Pull request by Syulin7**
- Fix the prompt content when submitting serve job.
- Default delete secret permissions in et-operator.
- Enable create secret for deepspeedjob, etjob.

### CODE REVIEW

File: pkg/apis/types/submit_deepspeedjob.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ type SubmitDeepSpeedJobArgs struct {
 	LauncherSelectors  map[string]string `yaml:"launcherSelectors"` // --launcher-selector
 	JobRestartPolicy   string            `yaml:"jobRestartPolicy"`  // --job-restart-policy
 	JobBackoffLimit    int               `yaml:"jobBackoffLimit"`   // --job-backoff-limit
+	// SSHSecret enables create secret for job.
+	SSHSecret  string            `yaml:"sshSecret"`
```
Comment (cheyang) [inline pkg/apis/types/submit_deepspeedjob.go:30]: SSHSecret -> SSHSecretName

File: pkg/argsbuilder/submit_deepspeedjob.go [hunk 2]
```diff
@@ -148,7 +152,9 @@ func (s *SubmitDeepSpeedJobArgsBuilder) check() error {
 
 func (s *SubmitDeepSpeedJobArgsBuilder) setEnv() error {
 	// avoid deepspeed job handing
-	s.args.Envs["NCCL_ASYNC_ERROR_HANDLING"] = "1"
+	if _, ok := s.args.Envs["NCCL_ASYNC_ERROR_HANDLING"]; !ok {
```
Comment (cheyang) [inline pkg/argsbuilder/submit_deepspeedjob.go:155]: Use constant to avoid duplication of NCCL_ASYNC_ERROR_HANDLING .

Comment (Syulin7) [reply pkg/argsbuilder/submit_deepspeedjob.go:155]: Done

File: install.sh [hunk 3]
```diff
@@ -256,15 +256,15 @@ function clean_old_env() {
...
-    if arena-kubectl get deployment,Service,ServiceAccount -n $NAMESPACE | grep kubedl-operator &> /dev/null;then
-        arena-kubectl delete deployment kubedl-operator -n $NAMESPACE
-        arena-kubectl delete ServiceAccount kubedl-operator -n $NAMESPACE
-        arena-kubectl delete crd crons.apps.kubedl.io
-        arena-kubectl delete ClusterRole kubedl-operator-role -n $NAMESPACE
-        arena-kubectl delete ClusterRoleBinding kubedl-operator-rolebinding -n $NAMESPACE
-        arena-kubectl delete Service kubedl-operator -n $NAMESPACE
-        arena-kubectl delete ServiceMonitor kubedl-operator -n $NAMESPACE
-    fi
+    #    if arena-kubectl get deployment,Service,ServiceAccount -n $NAMESPACE | grep kubedl-operator &> /dev/null;then
```
Comment (cheyang) [inline install.sh:259]: Please add warning before installing arena.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**